test/input.txt

```diff
@@ -148,6 +148,12 @@
   clear
   list
 
+##########################################################
+# test clear command
+##########################################################
+
+  repeat
```

Comment:
Perhaps you may want to add in a test case where the previous command before repeat is an invalid command.